src/test/kotlin/tech/harmonysoft/oss/gradle/release/paperwork/GradleReleasePaperworkPluginTest.kt

```diff
@@ -486,7 +486,28 @@ internal class GradleReleasePaperworkPluginTest {
         """.trimIndent())
         runBuild()
 
-        val expected = String.format(GradleReleasePaperworkPlugin.RELEASE_COMMIT_MESSAGE_PATTERN, version)
+        val expected = String.format(GradleReleasePaperworkPlugin.DEFAULT_RELEASE_COMMIT_MESSAGE_PATTERN, version)
+        val tag = git.tagList().call().find {
```

Comment:
it's not good to duplicate the code, it's better to extract it into a common place and reuse